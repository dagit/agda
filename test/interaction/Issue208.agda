module Issue208 where

record R : Set where
  foo : Set
  foo = {!!}