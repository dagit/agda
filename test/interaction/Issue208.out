(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-info-action "*Type-checking*" "Checking Issue208 (Issue208.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue208.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set\n" nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-status-action "")
(agda2-info-action "*Context*" "" nil)
((last . 1) . (agda2-goals-action '(0)))
